src/app/components/sections/job-comments-section/job-comments-section.component.ts

```diff
@@ -0,0 +1,21 @@
+import {Component} from '@angular/core';
+import {Input} from '@angular/core';
+import {Job} from '../../../models/api-models/job/job'
+
+@Component({
+  selector: 'job-comments-section',
+  template: `
+    <div class="ui basic center aligned segment">
+      <i class="ui big circular inverted pink wechat icon"></i>
```

Comment:
Is that the actual wechat logo? That could be quite misleading for a user that is familiar with the brand.. Plus we should use a generic icon and not a company logo ;)